src/runtime/testing/stub-loader.ts

```diff
@@ -8,44 +8,47 @@
  * http://polymer.github.io/PATENTS.txt
  */
 import {Loader} from '../loader.js';
+import {Dictionary} from '../hot.js';
 
 /**
  * A Loader initialized with a per-path canned responses.
  * Value for '*' key can be specified for a response if the path did not match.
  * If '*' is not specified and path is not matched, Loader logic is invoked.
  */
 export class StubLoader extends Loader {
-  _fileMap;
+  _fileMap: Dictionary<string>;
   _cannedResponse;
  
-  constructor(fileMap) {
+  constructor(fileMap: Dictionary<string>) {
     super();
     this._fileMap = fileMap;
     if (fileMap.hasOwnProperty('*')) {
       this._cannedResponse = fileMap['*'];
     }
   }
 
-  loadResource(path: string) {
+  // TODO(lindner): this should be async, but adding it causes a test to fail
```

Comment:
Changing this to async causes the _ exposes a dereference API to particles_ test to fail.  Guessing it's some timing related issue.